src/controller/postUrl.ts

```diff
@@ -0,0 +1,43 @@
+import express from "express";
+import { databaseModel } from "../model";
+import { nanoid } from "nanoid";
+import { logger } from "../services";
+import { response } from "../utils";
+
+export const postUrl = async (req: express.Request, res: express.Response) => {
+  try {
+    const { fullUrl, customUrl } = req.body;
```

Comment:
These should be the proper names of those request body properties according to the spec
```suggestion
    const { url, customName } = req.body;
```